karma/tests/layout/privacy.ts

```diff
@@ -91,4 +92,32 @@ describe("Layout: Privacy Tests", () => {
         done();
     }));
 
+    it("checks that node value is masked event when explicitly masked if unmask attribute is applied to parent",
+        testAsync(async (done: DoneFn) => {
+            watch();
+            let valueString = "value";
+            let maskedValueString = mask(valueString);
+            let skipParent = document.createElement("div");
+            let parent = document.createElement("div");
```

Comment:
Minor preference would be to just set the innerHTML and write this test as HTML fragment rather than a bunch of hard to read DOM calls.

Reply:
reasonable ask, but I'm going to punt refactoring our tests and let this go through as it's high pri

---
In reply to: [309715206](https://github.com/microsoft/clarity-js/pull/147#discussion_r309715206) [](ancestors = 309715206)